src/node/Node.ts

```diff
@@ -298,6 +298,9 @@ export class Node extends EventEmitter {
         if (this.destroyed) return;
         const json = JSON.parse(message as string);
         if (!json) return;
+        if (this.listeners('rawEvents').length) {
+            this.emit('rawEvents', json);
```

Comment:
Just do `this.emit('raw', json)`

Also don't forget to emit it from Shoukaku
You can follow this line of code for reference
https://github.com/Deivu/Shoukaku/blob/cfd4916df4b1882d81f0b3b711ef3f7d67c37904/src/Shoukaku.ts#L219

You don't usually listen to the node directly for events, you listen from the main class as listening to node *may* break your code